pkg/didmethod/peer/store.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package peerdid
```

Comment:
Generally the package name should match the folder name.

Reply:
Updated the package name (missed it during folder name refactoring).